flake.nix

```diff
@@ -35,6 +35,8 @@
           pnpm.configHook
         ];
 
+        STRIPE_API_KEY = "sk_test_...";
```

Comment:
This is a bit stupid, we're not doing this in the other CI and here as well...

Need to figure out a smarter way, perhaps just `cp .env.example .env`